frappe/integrations/doctype/google_settings/google_settings.json

```diff
@@ -18,10 +22,12 @@
    "label": "Enable"
   },
   {
+   "description": "The Client ID obtained from the Google Cloud Console under \"APIs & Services\" > \"Credentials\".",
```

Comment:
Can we do links/hyperlinks in the description? Might be useful to just link to https://console.cloud.google.com/apis/credentials for quick access.